sr_robot_launch/scripts/sr_ur_arm_unlock

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# Copyright 2020 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import rospy
+from std_msgs.msg import Bool
+from ur_dashboard_msgs.srv import GetSafetyMode, GetProgramState, GetRobotMode, Load
+from ur_dashboard_msgs.msg import SafetyMode, ProgramState, RobotMode
+from std_srvs.srv import Trigger
+
+class SrUrUnlock():
+    def __init__(self):
+        self.external_control_program_name = rospy.get_param("~urcap_program_name", "external_ctrl.urp")
+        self._arms = []
+        if rospy.has_param('ra_sr_ur_robot_hw'):
+            self._arms.append('ra')
+        if rospy.has_param('la_sr_ur_robot_hw'):
+            self._arms.append('la')
+        if 0 == len(self._arms):
+            rospy.logerr("No arms detected, shutting down %s", rospy.get_name())
+            rospy.signal_shutdown("No arms detected")
+        self.subscriber = rospy.Subscriber("/sr_arm/release", Bool, self.unlock_subscriber)
```

Comment:
No need to assign to an object

Reply:
Like this? ```suggestion
        rospy.Subscriber("/sr_arm/release", Bool, self.unlock_subscriber)
```

Reply:
yep

Reply:
369a9e027fa863b839d4b40ac8da5fe885be9d07